scrapi/harvesters/vivo.py

```diff
@@ -0,0 +1,216 @@
+                            ?dateURI vivo:dateTime ?date . 
+                            OPTIONAL {{
+                                        ?URI <http://purl.org/dc/terms/title> ?title .
+                                    }}
+                            OPTIONAL {{
+                                        ?journal vivo:publicationVenueFor ?URI .
+                                        ?journal rdfs:label ?journalTitle .
+                                    }}
+                            OPTIONAL {{
+                                        ?journal vivo:publicationVenueFor ?URI .
+                                        ?journal bibo:issn ?ISSN .
+                                    }}
+                            OPTIONAL {{
+                                        ?journal vivo:publicationVenueFor ?URI .
+                                        ?journal bibo:isbn ?ISBN .
+                                    }}
+                            OPTIONAL {{ 
+                                        ?publisherURI vivo:publisherOf ?journal .
+                                        ?publisherURI rdfs:label ?publisher .
+                                     }}
+                            OPTIONAL {{ 
+                                        ?subjectURI vivo:subjectAreaOf ?URI .
+                                        ?subjectURI rdfs:label ?subject .
+                                    }}
+                            OPTIONAL {{ 
+                                        ?URI vivo:freetextKeyword ?keyword .
+                                    }}
+                            OPTIONAL {{
+                                        ?URI vitro:mostSpecificType ?typeURI .
+                                        ?typeURI rdfs:label ?type .
+                                    }}
+                            OPTIONAL {{
+                                        ?URI bibo:abstract ?abstract .
+                                    }}        
+                            OPTIONAL {{
+                                        ?URI bibo:doi ?doi .
+                                    }}
+                            OPTIONAL {{
+                                        ?URI bibo:volume ?volume .
+                                    }}
+                            OPTIONAL {{
+                                        ?URI bibo:number ?number .
+                                    }}
+                            OPTIONAL {{
+                                        ?URI bibo:pageStart ?startPage .
+                                    }}
+                            OPTIONAL {{
+                                        ?URI bibo:pageEnd ?endPage .
+                                    }}
+                            OPTIONAL {{
+                                        ?URI bibo:pmid ?PMID .
+                                   }} 
+                            OPTIONAL {{
+                                        ?autorship a vivo:Authorship .
```

Comment:
is `autorship` the correct name, or should it be `authorship`? I am not familiar with the API, so I am not sure if this is a typo or not.